sql/core/src/main/scala/org/apache/spark/sql/GroupedData.scala

```diff
@@ -46,6 +46,15 @@ class GroupedData protected[sql](df: DataFrameImpl, groupingExprs: Seq[Expressio
     }
   }
 
+  @scala.annotation.varargs
+  private[this] def aggregateNumericColumns(colName: String, colNames: String*)
```

Comment:
can you just remove the other aggregateNumericColumns and leave this one?

Also for this one there is no need to use the annotation because it is not a public API.